quam_builder/architecture/quantum_dots/exceptions.py

```diff
@@ -0,0 +1,32 @@
+# --- Custom Exceptions ---
```

Comment:
Why having these exceptions here and not in voltage_gate_sequence.py?

Reply:
I moved it to the voltage_sequence folder.
I don't think it makes sense to add to the file itself as it's also used in other files

Reply:
Alright, I am not very familiar with the idea of having a separated file for Exceptions, but I trust you :)